docs/tutorials/portable-consent.md

```diff
@@ -0,0 +1,324 @@
+---
+sidebar_label: Portable consent
+sidebar_position: 5.5
+hide_table_of_contents: false
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import requeststab from '/docs/tutorials/img/requests-tab.png';
+import messagestab from '/docs/tutorials/img/messages-tab.png';
+import consentlogic from '/docs/tutorials/img/consent-state-logic.png';
+
+# Use portable consent for XMTP
+
+![Feature status](https://img.shields.io/badge/Feature_status-Alpha-orange)
+
+:::caution This feature is in **alpha** status
+
+This feature is in **alpha** status and ready for you to start experimenting with. However, we do not recommend using alpha features in production apps. Expect frequent changes as we iterate based on feedback.
+
+:::
+
+When you build an app with XMTP, you can choose to implement portable consent. 
+
+With portable consent, a blockchain account address registered on the XMTP network can have one of three contact consent states:
+
+- Unknown
+- Allowed
+- Denied
+
+Contact consent states are stored in a consent list on the XMTP network. The consent list is accessible by all apps.
+
+## Understand contact consent states
+
+Here are some of the ways contact consent states are set:
+
+### Unknown
+
+Conversation created using an app **with** portable consent implemented:
+
+- For a new conversation that a peer contact wants to start with a user, the contact consent state is set to `unknown`.
+
+Conversation created using an app **without** portable consent implemented:
+
+- For an existing conversation created by a peer contact and that a user hasn't responded to, the contact consent state is set to `unknown`.
+
+### Allowed
+
+Conversation created using an app **with** portable consent implemented:
```

Comment:
Not necessarily about implemented but the app is on a sdk version that supports consent. Since this is implicit without needed action by the developer.